src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -760,7 +843,11 @@ static ScenarioGraph buildGraph(const std::vector<OpDesc>& op_descs,
 static StreamDesc parseAdvancedStream(const YAML::Node& node, const GlobalOptions& opts,
```

Comment:
validation in `parseStream()` that stands for old way of pipeline specification is omitted

Reply:
Nothing done. Sorry don't understand.